pkgs/development/python-modules/rich/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, isPy27, typing-extensions, pygments, recommonmark, colorama }:
+
+buildPythonPackage rec {
+  pname = "rich";
+  version = "8.0.0";
+  disabled = isPy27;
```

Comment:
The difference is probably minimal, but `pythonOlder "3.6"` is probably more accurate? https://github.com/willmcgugan/rich/blob/master/pyproject.toml#L26